scripts/prepdocslib/filestrategy.py

```diff
@@ -50,13 +43,14 @@ async def setup(self, search_info: SearchInfo):
             search_info,
             self.search_analyzer_name,
             self.use_acls,
+            False,
             self.embeddings,
             search_images=self.image_embeddings is not None,
         )
         await search_manager.create_index()
 
     async def run(self, search_info: SearchInfo):
-        search_manager = SearchManager(search_info, self.search_analyzer_name, self.use_acls, self.embeddings)
+        search_manager = SearchManager(search_info, self.search_analyzer_name, self.use_acls, False, self.embeddings)
```

Comment:
Is there a reason we don't reuse the manager created in setup?

Reply:
This code has been for a while. Need to be cleaned up.